client/packages/common/src/hooks/useUrlQuery/useUrlQueryParams.ts

```diff
@@ -22,7 +22,9 @@ export const useUrlQueryParams = ({
   // do not coerce the filter parameter if the user enters a numeric value
   // if this is parsed as numeric, the query param changes filter=0300 to filter=300
   // which then does not match against codes, as the filter is usually a 'startsWith'
-  const { urlQuery, updateQuery } = useUrlQuery({ skipParse: ['filter'] });
+  const { urlQuery, updateQuery } = useUrlQuery({
+    skipParse: [filterKey ?? 'filter'],
```

Comment:
prevent the `useUrlQuery` from parsing the filter values as numbers or dates. strings only for filters.